tests/components/zwave_mqtt/test_binary_sensor.py

```diff
@@ -0,0 +1,66 @@
+"""Test Z-Wave Sensors."""
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_MOTION,
+    DOMAIN as BINARY_SENSOR_DOMAIN,
+)
+from homeassistant.components.zwave_mqtt.const import DOMAIN
+from homeassistant.const import ATTR_DEVICE_CLASS
+
+from .common import setup_zwave
+
+
+async def test_binary_sensor(hass, generic_data, binary_sensor_msg):
+    """Test setting up config entry."""
+    receive_msg = await setup_zwave(hass, fixture=generic_data)
+
+    # Test Legacy sensor (disabled by default)
+    registry = await hass.helpers.entity_registry.async_get_registry()
+    entity_id = "binary_sensor.trisensor_sensor"
+    state = hass.states.get(entity_id)
+    assert state is None
+    entry = registry.async_get(entity_id)
+    assert entry
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)